app/controllers/menus_controller.rb

```diff
@@ -4,8 +4,11 @@ def new
   end
 
   def create
-    @menu = Menu.new(menu_params)
+    @restaurant = Restaurant.find(current_user.restaurants.first)
+    @menu = @restaurant.menus.build(menu_params)
+    binding.pry
```

Comment:
Aware that I left 'PRY'... Had more errors to sort so pry was in there for debugging...